subscriptions/vscloud-csp.md

```diff
@@ -27,11 +27,12 @@ You must first set up your customer tenant in the Partner Center and create an A
 [Learn more](/azure/devops/organizations/billing/csp/set-up-csp-customer)
 
 ## Who can buy Visual Studio subscriptions?
+
 Anyone with [owner or contributor access](https://na01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fdocs.microsoft.com%2Fen-us%2Fvsts%2Forganizations%2Fbilling%2Fadd-backup-billing-managers%3Fview%3Dvsts%2520%2520sa&data=02%7C01%7C%7Cb9e717e8abff47b0cd7e08d618edd860%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636723807145220358&sdata=aIaamEXHhx94KCYVY%2FFibqFzNBEqKPntpql867xAMgU%3D&reserved=0) to the Azure subscription can purchase Visual Studio subscriptions.
 
 ## How to buy
 
-<iframe src="//channel9.msdn.com/Shows/Visual-Studio-for-CSP-Partners/CSP-How-to-buy-Visual-Studio-Subscriptions/player" width="600" height="315" allowFullScreen="true" frameBorder="0"></iframe>
+> [!VIDEO https://channel9.msdn.com/Shows/Visual-Studio-for-CSP-Partners/CSP-How-to-buy-Visual-Studio-Subscriptions/player]
```

Comment:
@evanwindom Any reason not to make this change in this file?

Reply:
Honestly, I don't know.  James authored that article.

Reply:
I'd suggest that we go ahead and publish the change. It's correct Markdown syntax for a video from Channel 9.